tests/x11regressions/gnote/gnote_link_note.pm

```diff
@@ -28,7 +28,8 @@ sub run {
     send_key 'ctrl-ret';    #switch to link
     assert_screen 'gnote-note-start-here';
 
-    wait_screen_change { send_key 'ctrl-tab' };    #jump to toolbar
+    wait_screen_change { send_key 'ctrl-tab' };                                    #jump to toolbar
+    wait_screen_change { send_key 'ctrl-tab' } if (sle_version_at_least('15'));    #jump to toolbar
```

Comment:
Same as above, `is_sle` is needed to test a SLE version.

Reply:
Fixed, please help to review.